Makefile

```diff
@@ -78,6 +79,10 @@ lint-errors: images
 
 tests: tests7 tests8
```

Comment:
We should add the `tests6` here as well, so when we execute the whole `make tests` command, we get everything executed. 

But this is not a blocker for me. We can merge this and add just this piece of code in another PR.

Reply:
Thanks for the clarification!